lesson-17/index.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+
+// HW-17
+
+// task 1
+
+// const a = ("'true'");
+// const b = Boolean(NaN);
+// const c = (17);
+// const d = undefined;
+// const e = null;
+// console.log(typeof a);
+// console.log(typeof b);
+// console.log(typeof c);
+// console.log(typeof d);
+// console.log(typeof e);
+
+
+// task 2
+
+// const height = 15;
+// const width = 20;
+
+// if (height > width) {
+//     console.log(height);
+// } else if (width > height) {
+//     console.log(width);
+// }
+
+
+// task 3
+
+// for (let i = 1; i <= 20; i+=1) {
+//     if (i % 3 == 0) {
+//       console.log(i);
+//     }
+// }
+
+
+// task 4
+
+// const key = true;
+// const documents = true;
+// const pen = true;
+// const apple = false;
+// const orange = true;
+// let shouldGoToWork = (key && documents && pen && apple || orange);
+// console.log(shouldGoToWork);
+
+
+// task 5
+
+// let userNumber = prompt("Введите число");
+
+// if (userNumber % 3 === 0 && userNumber % 5 === 0) {
+//     console.log("FizBuz");
+// } else if (userNumber % 5 === 0) {
+//     console.log("Fiz");
+// } else if (userNumber % 3 === 0) {
+//     console.log("Buz");
+// }
+
+
+// task 6
+
+// let age = prompt("Введите возраст");
+
+// if (age >= 18) {
+//     alert("Попей пивка");
+//     alert("Можешь выкурить сигаретку, только маме не говори");
+// } else if(age < 18) {
+//     alert("Пей колу");
+//         if(age >= 16) { 
+//             alert("Можешь выкурить сигаретку, только маме не говори");
+//         }
+// } 
```

Comment:
В задании написано, что мы не включаем 18, то есть условие не совсем корректно. Также не нужный отступ во внутреннем условии
```suggestion
// if (age > 18) {
//     alert("Попей пивка");
//     alert("Можешь выкурить сигаретку, только маме не говори");
// } else if(age <= 18) {
//     alert("Пей колу");
//     if(age >= 16) { 
//        alert("Можешь выкурить сигаретку, только маме не говори");
//     }
// } 
```